### DISCUSSION TITLE
生成切片速度缓慢，是否是我电脑配置或者设置问题

### DISCUSSION THREAD

**Discussion by CangMoJian** [Xinrea/bili-shadowreplay#22] [Q&A]
在主播直播过程当中利用快捷窗口进行了一个两分钟的切片，完成选区后点击了“从选区生成新切片”，然后开始等待，经过了二十分钟等待依旧未能完成。不知道是否是我笔记本电脑配置影响了效率，亦或是我的电脑设置有什么不对。以前没有使用过录制和剪辑软件的经验，属于新手。

**Comment by Xinrea** [comment]
正常不会耗时这么久，可能是生成失败卡住了；可以刷新后选取其它长度的片段试试是否也会有问题。
如果都失败了，可以尝试从命令行运行程序，然后使用切片功能，看看命令行中的日志是否有记录到任何错误。
具体可以这样操作：
- 找到程序的安装目录，按住 shift 点击鼠标右键，可以看到右键菜单中出现了 `在此处打开 Powershell` 窗口，选择它出现命令行窗口。
- 在命令行窗口中输入 `./bili-shadowreplay.exe`，程序会正常启动，同时窗口里面会有日志记录出现。

或者等今天晚点，将会更新一个新版本，新版本能够提供日志文件，能更好地定位问题。

**Reply by Xinrea** [reply]
https://github.com/Xinrea/bili-shadowreplay/releases/tag/v1.1.0
新版本添加了日志输出，同时修改了下切片编码方式，如果还有问题可以附上日志文件。
